website/src/routes/apiv2.js

```diff
@@ -0,0 +1,776 @@
+/*
```

Comment:
not directly, there's require('../../../common').content but that was migrated to ES6 right? Also tests are passing